packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -131,7 +131,7 @@ function FlowsheetDataGrid(props) {
   const inShiftDirectionalMode = useRef(false);
   const flowsheetColumns = useMemo(() => columns.map(column => ({
     ...column,
-    isSelectable: true,
+    isSelectable: false,
```

Comment:
Do we need to update any wdio tests for this change?

Reply:
Yes good catch. It did cause some slight visual changes, such as the hover behavior and the focus indicator. I am generating those now

Reply:
Done